### ISSUE TITLE
Allow use of curl instead of wget for build

### ISSUE THREAD

**Issue by heitbaum** [libretro/scummvm#67]
Could the build scripts be updated to optionally allow curl instead of wget
- https://github.com/libretro/scummvm/blob/master/backends/platform/libretro/scripts/bundle_datafiles.sh
- https://github.com/libretro/scummvm/blob/master/dists/emscripten/build.sh

```
--- a/backends/platform/libretro/scripts/bundle_datafiles.sh    2024-02-12 22:13:53.000000000 +0000
+++ b/backends/platform/libretro/scripts/bundle_datafiles.sh    2024-10-20 11:20:58.695910358 +0000
@@ -108,7 +108,12 @@
 if [ ! $3 = "bundle" ]; then
 
 # Update from libretro ScummVM.dat
-wget -NO "$BUILD_PATH"/ScummVM.dat https://raw.githubusercontent.com/libretro/libretro-database/master/dat/ScummVM.dat
+if command -v wget >/dev/null; then
+  wget -NO "$BUILD_PATH"/ScummVM.dat https://raw.githubusercontent.com/libretro/libretro-database/master/dat/ScummVM.dat
+else
+  # if wget is not available use curl
+  curl -f -o "$BUILD_PATH"/ScummVM.dat https://raw.githubusercontent.com/libretro/libretro-database/master/dat/ScummVM.dat
+fi
 [ -f "$BUILD_PATH"/ScummVM.dat ] && SUPPORTED_EXTENSIONS="$(cat $BUILD_PATH/ScummVM.dat | grep 'rom (' | sed -e 's/\" .*//g' -e 's
/.*\.//g' | sort -u | tr '\n' '|')" || SUPPORTED_EXTENSIONS="$ALLOWED_EXT"
 
        # Create core.info file
```

**Comment by spleen1981** [comment]
Can you submit a PR to the [staging_master](https://github.com/libretro/scummvm/tree/staging_master) branch for the patch above?
Not sure why you are mentioning also `dists/emscripten/build.sh`, but any change to that should be submitted to the upstream [scummvm/scummvm](https://github.com/scummvm/scummvm) repo.

**Comment by heitbaum** [comment]
> Can you submit a PR to the [staging_master](https://github.com/libretro/scummvm/tree/staging_master) branch for the patch above? Not sure why you are mentioning also `dists/emscripten/build.sh`, but any change to that should be submitted to the upstream [scummvm/scummvm](https://github.com/scummvm/scummvm) repo.

Thanks. I think only bundle_datafiles.sh is required. I’ll raise the PR shortly.

**Comment by spleen1981** [comment]
merged in master 8dffadf6724b3f0ec98ec10154f26e5f3cc98962
